src/registrar/abha-components/abha-enter-otp-component/abha-enter-otp-component.component.ts

```
@@ -289,6 +289,27 @@ export class AbhaEnterOtpComponentComponent {
     }, 1000);  // Update every second
   }
 
+  //Method to check if generated ABHA has any beneficiary ID linked
+  getLinkedBenIdsToAbha(data : any) {
+    let reqObj = {
+      healthIdNumber: data.healthIdNumber
+    }
+    this.registrarService.fetchBenIdLinkedToAbha(reqObj).subscribe((res: any) => {
+      if (res && res.data && res.data) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)